be/src/common/config.cpp

```diff
@@ -1598,6 +1598,9 @@ DEFINE_mBool(print_stack_when_cache_miss, "false");
 
 DEFINE_mBool(read_cluster_cache_opt_verbose_log, "false");
 
+DEFINE_mBool(enable_custom_aws_credentials_chain, "true");
```

Comment:
use string to choose aws_credentials_provider_chain_version = "v2"

Reply:
> use string to choose aws_credentials_provider_chain_version = "doris_custom_provider_chain_1" candidates can be: aws_default_provider_chain, doris_custom_provider_chain_0, doris_custom_provider_chain_1...

why has 3 options "aws_default_provider_chain, doris_custom_provider_chain_0, doris_custom_provider_chain_1" ?